content/assignments/candidate-testing/running-tests/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Final Steps"
+date: 2023-06-20T10:39:25-05:00
+draft: false
+weight: 5
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Congratulations!  
+
+You have completed Graded Assignment 1!  
+Save, stage, commit, and push this up to your repo.
+
+When your repo receives a green check mark, let your TA know.  In order to be marked complete, you will walk your TA through your code.
+
+Your repository will run the tests ANY time you push and on any branch.
+
+## Run the Tests
```

Comment:
I've made note in Parts 1 and 2 about adding in a note of how many tests would pass if the student did a git add, commit and push after completing each part either by checking github actions or running npm test locally.  Suggest changing the wording to reflect more of a if you haven't run the test or looked at actions to do so now.